server/util/twilio.js

```diff
@@ -44,12 +44,20 @@ export function sayCallCompleted() {
   return CallCompleted.toString();
 }
 
+export function hangUpContactCall(callSid, user_id) {
+  return twilioClient.calls(callSid)
+    .update({
+      status: 'completed',
+      statusCallback: `${DEV_PATH}/users/${user_id}/bridge`
```

Comment:
No - this is not how we're supposed to hang up the call. We hang it up by changing the call's url to the bridge url. Changing the status to completed will actually terminate the call with the volunteer! And we don't want to change the statusCallback either. Does this make sense?

Reply:
hmmm kind of, I'll look back at the docs and see what the other props are I can choose from.